include/accessor.h

```diff
@@ -221,17 +222,21 @@ class accessor<DataT, Dims, Mode, target::device> : public detail::accessor_base
 	friend struct detail::accessor_testspy;
 
   public:
+	CELERITY_DETAIL_HACK_CLANG_TIDY_ALLOW_NON_CONST(accessor)
+	accessor(accessor&&) noexcept = default;
+	accessor& operator=(accessor&&) noexcept = default;
```

Comment:
Consider moving the move-assign definition closer to the copy-assign below (it is not related to the `CLANG_TIDY_HACK`)